website/blog/2025-11-24-aks-automatic-managed-system-node-pools/index.md

```diff
@@ -0,0 +1,172 @@
+---
+title: "Announcing AKS Automatic managed system node pools (preview)"
+description: "Discover how AKS Automatic now offers Microsoft-managed system node pools so you can ship apps faster with zero infrastructure overhead, built-in add-on reliability, and optimized costs."
+date: 2025-11-24
+authors: [ahmed-sabbour]
+tags:
+  - aks-automatic
+  - managed-system-node-pools
+keywords: ["AKS", "Kubernetes", "Automatic", "Managed system node pools"]
+---
+
+ We built Azure Kubernetes Service (AKS) Automatic so teams can ship applications with production-grade defaults from day one. Today we're taking that vision further with **managed system node pools (preview)**: the system pool is now fully managed by AKS, with core platform components hosted on Microsoft infrastructure.
+
+<!-- truncate -->
+
+> Learn more in the official documentation: [Managed system node pools on AKS Automatic (preview)](https://learn.microsoft.com/azure/aks/automatic/aks-automatic-managed-system-node-pools-about)
+
+## Why it matters
+
+- **Reduced operational overhead:** AKS handles provisioning, patching, upgrades, and scaling for the system pool, so you spend less time on infrastructure maintenance.
+- **Managed add-on hosting at lower cost:** Core services like Azure Monitor agents, CoreDNS, KEDA, VPA, Konnectivity, Eraser, and Metrics Server run on Microsoft-owned infrastructure. Some add-ons and `DaemonSets` still run on nodes in your subscription.
+- **Built-in security policies:** Deployment Safeguards enforce pod security standards, restrict access to platform namespaces, and block risky configurations by default.
+- **Automatic upgrades:** AKS keeps platform components current, reducing the risk of running outdated or vulnerable system software.
+
+## How managed system node pools differ from traditional system node pools
+
+| Aspect | AKS Standard system pool | AKS Automatic managed system pool |
+| --- | --- | --- |
+| **Provisioning** | You create the pool, select VM SKUs, set node count, and configure OS disk size | AKS provisions and sizes the pool for you automatically |
+| **Capacity planning** | You estimate headroom for system components like CoreDNS, konnectivity, metrics-server, and any add-ons; scale manually or configure cluster autoscaler with min/max counts | AKS right-sizes capacity for platform components and scales automatically when add-ons need more room |
+| **Cost** | System nodes are billed as standard VMs to your subscription; you pay for system pool capacity | System nodes do not run on your subscription |
+
+![Choose your operating model between AKS Standard and AKS Automatic](aks-standard-automatic.png)
+
+## Components running on managed system node pools

```

Comment:
Think this section needs to come before others.  Esp as you show conceptually what is a system nodepool and what areits components come organically next.